src/ui/CookiesPopup.svelte

```diff
@@ -55,6 +69,10 @@
     right: 0;
     z-index: 9999;
     text-align: center;
+
+    :global(body:not(.desktop)) & {
+      padding: 24px 20px;
+    }
```

Comment:
Desktop's styles are defined on `LOC 106`. `:global(body:not(.desktop))` is redundant